src/sparsematrix.jl

```diff
@@ -1135,6 +1135,37 @@ sparse(I,J,V::AbstractVector{Bool},m,n) = sparse(I, J, V, Int(m), Int(n), |)
 
 sparse(I,J,v::Number,m,n,combine::Function) = sparse(I, J, fill(v,length(I)), Int(m), Int(n), combine)
 
+## Explicit comparisons with transposed arrays
+
+# Check whether all nonzero elements of A are equal to the respective elements in B
+function nzeq(A::SparseMatrixCSC, B::Union{Adjoint{<:Any,<:AbstractSparseMatrixCSC},Transpose{<:Any,<:AbstractSparseMatrixCSC}})
```

Comment:
Should we use the `AdjOrTrans` type here?

Reply:
PS. it's in LinearAlgebra (but it's not exported)

Reply:
It's not used elsewhere in `SparseArrays.jl` either, and there are a few places where it could be used.

I am now using `AbstractSparseMatrixCSCInclAdjointAndTranspose` instead where possible.

Reply:
I am now defining a new type `AbstractSparseMatrixCSCInclNestedAdjointAndTranspose`. Note the nested `Nested` in here.